docs/pages/get-started.mdx

```diff
@@ -106,10 +106,18 @@ allowed to log into SSH hosts as any of the principals `root`, `ubuntu`, or
 
 On your Linux host, run the following command:
 
+<ScopedBlock scope={["oss"]}>
```

Comment:
This page is intended only for OSS users, so I don't think we need the `ScopedBlock` here